vendor/github.com/openshift/library-go/pkg/config/helpers/config_refs.go

```diff
@@ -0,0 +1,99 @@
+package helpers
+
+import (
+	"strings"
+
+	configv1 "github.com/openshift/api/config/v1"
+)
+
+func GetHTTPServingInfoFileReferences(config *configv1.HTTPServingInfo) []*string {
```

Comment:
why `*string` instead of string?

Reply:
> why *string instead of string?

You need a pointer to the string so you can mutate the original in place.